applications/crossbar/src/modules/cb_recordings.erl

```diff
@@ -68,8 +64,8 @@ init() ->
 -spec allowed_methods(path_token(), path_token()) -> http_methods().
 
 allowed_methods() -> [?HTTP_GET].
-allowed_methods(_RecordingId ) -> [?HTTP_GET].
-allowed_methods(_RecordingId, ?ATTACHMENT) -> [?HTTP_GET].
+allowed_methods(_) -> [?HTTP_GET].
+allowed_methods(_, _) -> [?HTTP_GET].
```

Comment:
> by removing _RecordingId the generated api doc is ID instead of RECORDING_ID. any reason for trying to do that ?
@lazedo Hi Luis, No reasons. Just didn't know why bind to _VariableName. Thanks!